packages/robot-hooks/test/test.js

```diff
@@ -133,12 +133,19 @@ QUnit.module('useMachine', hooks => {
     el.remove();
   });
 
-  QUnit.skip('Invoking machines the "current" is the child machine', async assert => {
+  QUnit.test('Invoking machines the "current" is the child machine', async assert => {
+    const deep = createMachine({
+      deepOne: state(
+        transition('next', 'deepTwo')
+      ),
+      deepTwo: state(),
+    })
```

Comment:
missing semi-colon? (the following `const nested` *is* semi-colon terminated)

Reply:
```suggestion
    });
```